src/utils/linkResolver.js

```diff
@@ -1,21 +1,23 @@
-import { defaultLanguage } from '../../prismic-config'
+import { defaultLanguage } from "../../prismic-config";
```

Comment:
Is there a reason you changed these to double quotes?

Reply:
No particular reason.